server/src/main/scala/za/co/absa/atum/server/api/database/DoobieImplicits.scala

```diff
@@ -20,19 +20,37 @@ import cats.Show
 import cats.data.NonEmptyList
 import doobie.{Get, Put}
 import doobie.postgres.implicits._
+import io.circe.{Json => CirceJson}
 import org.postgresql.jdbc.PgArray
 import org.postgresql.util.PGobject
 
 import scala.util.{Failure, Success, Try}
 
 object DoobieImplicits {
 
-  private implicit val showPGobject: Show[PGobject] = Show.show(_.getValue.take(250))
   private implicit val showPgArray: Show[PgArray] = Show.fromToString
 
   implicit val getMapWithOptionStringValues: Get[Map[String, Option[String]]] = Get[Map[String, String]]
       .tmap(map => map.map { case (k, v) => k -> Option(v) })
 
+  private def circeJsonListToPGJsonArrayString(jsonList: List[CirceJson]): String = {
```

Comment:
this doesn't work... I am currently testing these implicits in fa-db

Reply:
Okay, I'm checking it.